applications/CableNetApplication/custom_elements/sliding_cable_element_3D.cpp

```diff
@@ -318,6 +318,7 @@ Vector SlidingCableElement3D::GetDirectionVectorNt() const
 
 Vector SlidingCableElement3D::GetInternalForces()
 {
+  const double numerical_limit = std::numeric_limits<double>::epsilon();
```

Comment:
This can defined as as constexpr

Reply:
Why `std::numeric_limits<double>::epsilon`?

Reply:
also, consistency with the other elements.